src/components/List/ListItem/ListItem.jsx

```diff
@@ -120,7 +124,7 @@ const ListItem = ({
     return nestingLevel > 0 ? (
       <div
         className={`${iotPrefix}--list-item--nesting-offset`}
-        style={{ width: `${nestingLevel * 30}px` }}
+        style={{ width: `${(nestingLevel - 1) * 30}px` }}
```

Comment:
This causes a regression with indentation in hierarchy lists.

![image](https://user-images.githubusercontent.com/3360588/99279133-7f3b5e00-27f5-11eb-9a80-daa1b66de4fb.png)
 `My Reports` should be indented to be aligned with the `Tasks`